packages/vuetify/src/components/VDataTable/VSimpleTable.ts

```diff
@@ -12,6 +12,7 @@ export default mixins(Themeable).extend({
     dense: Boolean,
     fixedHeader: Boolean,
     height: [Number, String],
+    tableClasses: Object,
```

Comment:
Should have a default value

```
tableClasses: {
  type: Object,
  default: () => ({}),
}
```

Reply:
Tried move the mobile css to tr at MobileRow